src/sentry/notifications/platform/email/provider.py

```diff
@@ -34,3 +35,13 @@ class EmailNotificationProvider(NotificationProvider[EmailRenderable]):
     @classmethod
     def is_available(cls, *, organization: RpcOrganizationSummary | None = None) -> bool:
         return True
+
+    @classmethod
+    def _send_notification(
+        cls,
+        *,
+        notification_content: EmailRenderable,
+        notification_type: NotificationType,
+        target: NotificationTarget,
+        thread_id: str | None = None,
+    ) -> None: ...
```

Comment:
Need to add doc comments, but this should handle actually sending the notification to the provider.